src/MapzenSearchDataObjects.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  MapzenRequestConfig.swift
+//  Pods
+//
+//  Created by Sarah Lensing on 3/20/17.
+//
+//
+
+import Foundation
+import Pelias
+
+public class MzSearchRect: NSObject {
```

Comment:
Yeah like I said above we can probably remove all of the prefixes here and then for things like `SearchBoundaryCircle` which will conflict on the namespace you should be able to reference it by module name like `Pelias.SearchBoundaryCircle`